src/development/metrics.md

```diff
@@ -0,0 +1,153 @@
+# Metrics
+
+This page gather all the measurement metrics, so that we can easily refer to them in the future. It is broken into four parts:
+
+ - The measurements that are common to both verkle and binary trees
+ - The measurements that are specific to verkle trees
+ - The measurements that are specific to binary trees
+ - The measurements that are specific to state expiry
+
+This is a living document, that we will fill as questions arise and we gather more data.
+
+##  Common measurements
+
+### Execution performance
+
+#### impact of key hashing
+
+In order to estimate the performance impact of Pedersen key hashing, this is a comparison of how long it takes to replay 200k blocks using Pedersen hashes, vs. sha256:
+
+![image.png](./assets/hash_speed_diff.png)
+
+At the end of the sha256 run, so >200k blocks later, the chain was 75072 blocks ahead. This represents a loss of 3 hours per day against sha256. That’s >12% faster.
+
+Coupled to the fact that Pedersen hashes aren’t quantum resistant, it seems like a good idea to reconsider using Pedersen hashes to compute trees.
+
+## Verkle-specific measurements
```

Comment:
BTW, I think the previous part should actually be part of this section since you're trying to measure Pedersen hashes which is VKT related.